docs/website/docs/tutorial/rest-api.md

```diff
@@ -0,0 +1,320 @@
+
+Let's update our example to use the `merge` write disposition. We need to specify the primary key for the `pokemon` resource and set the write disposition to `merge`:
+
+```py
+...
+pokemon_source = rest_api_source(
+    {
+        "client": {
+            "base_url": "https://pokeapi.co/api/v2/",
+        },
+        "resource_defaults": {
+            "endpoint": {
+                "params": {
+                    "limit": 1000,
+                },
+            },
+            # For the `berry` and `location` resources, we set
+            # the write disposition to `replace`
+            "write_disposition": "replace",
+        },
+        "resources": [
+            {
+                "name": "pokemon",
+                "primary_key": "id", # Setting the primary key for the `pokemon` resource
+                "write_disposition": "merge",
+            },
+            "berry",
+            "location",
+        ],
+    }
+)
+```
+
+## Loading data incrementally
+
+When working with some APIs, you may need to load data incrementally to avoid fetching the entire dataset every time and to reduce the load time. The API that support incremental loading usually provide a way to fetch only new or changed data (most often by using a timestamp field like `updated_at`, `created_at`, or incremental IDs).
+
+To illustrate incremental loading, let's consider the GitHub API. In the `rest_api_pipeline.py` file, you can find an example of how to load data from the GitHub API incrementally. Let's take a look at the configuration:
+
+```py
+pipeline = dlt.pipeline(
+    pipeline_name="rest_api_github",
+    destination="duckdb",
+    dataset_name="rest_api_data",
+)
+
+github_source = rest_api_source({
+    "client": {
+        "base_url": "https://api.github.com/repos/dlt-hub/dlt/",
+    },
+    "resource_defaults": {
+        "primary_key": "id",
+        "write_disposition": "merge",
+        "endpoint": {
+            "params": {
+                "per_page": 100,
+            },
+        },
+    },
+    "resources": [
+        {
+            "name": "issues",
+            "endpoint": {
+                "path": "issues",
+                "params": {
+                    "sort": "updated",
+                    "direction": "desc",
+                    "state": "open",
+                    "since": {
+                        "type": "incremental",
+                        "cursor_path": "updated_at",
+                        "initial_value": "2024-01-25T11:21:28Z",
+                    },
+                },
+            },
+        },
+    ],
+})
+
+load_info = pipeline.run(github_source())
+print(load_info)
+```
+
+In this configuration, the `since` parameter is defined as an special incremental parameter. The `cursor_path` field specifies the JSON path to the field that will be used to fetch the updated data and we use the `initial_value` for the initial value for the incremental parameter. This value will be used in the first request to fetch the data.
```

Comment:
```suggestion
In this configuration, the `since` parameter is defined as a special incremental parameter. The `cursor_path` field specifies the JSON path to the field that will be used to fetch the updated data and we use the `initial_value` for the initial value for the incremental parameter. This value will be used in the first request to fetch the data.
```